### PR TITLE
Add static preview to some [GitHub] services

### PR DESCRIPTION
**Pull request by paulmelnikow**

### CODE REVIEW

File: services/github/github-issues.service.js [hunk 1]
```diff
@@ -25,81 +25,194 @@ module.exports = class GithubIssues extends LegacyService {
...
+        },
+        keywords: ['issue'],
         documentation,
       },
       {
-        title: 'GitHub pull requests',
-        previewUrl: 'issues-pr/cdnjs/cdnjs',
-        keywords: ['GitHub', 'pullrequest', 'pr'],
+        title: 'GitHub issues by-label',
+        pattern: 'issues-raw/:user/:repo/:label',
```
Comment (calebcartwright) [inline services/github/github-issues.service.js]: should this begin with `issues/:user/...`? if not what's the difference between this example and the next one aside from the value of the label being used?

File: services/github/github-issues.service.js [hunk 2]
```diff
@@ -25,81 +25,194 @@ module.exports = class GithubIssues extends LegacyService {
...
+        },
+        keywords: ['pullrequest', 'pr'],
         documentation,
       },
       {
-        title: 'GitHub issues by-label',
-        previewUrl: 'issues-raw/badges/shields/service-badge',
-        keywords: ['GitHub', 'issue', 'label'],
+        title: 'GitHub closed pull requests',
+        namedParams: {
```
Comment (calebcartwright) [inline services/github/github-issues.service.js]: think we're missing a `pattern` on this example

File: services/github/github-languages.service.js [hunk 3]
```diff
@@ -33,8 +33,16 @@ class GithubCodeSize extends LegacyService {
     return [
       {
         title: 'GitHub code size in bytes',
-        previewUrl: 'badges/shields',
-        keywords: ['GitHub', 'byte', 'code', 'size'],
+        pattern: ':user/:repo',
```
Comment (calebcartwright) [inline services/github/github-languages.service.js]: Could we add this `pattern` to the route definition above?

File: services/github/github-size.service.js [hunk 4]
```diff
@@ -25,15 +25,26 @@ module.exports = class GithubSize extends LegacyService {
...
     }
   }
 
   static get examples() {
     return [
       {
-        title: 'GitHub repo size in bytes',
-        previewUrl: 'webcaetano/craft/build/phaser-craft.min.js',
-        keywords: ['GitHub', 'file', 'size'],
+        title: 'GitHub file size in bytes',
```
Comment (calebcartwright) [inline services/github/github-size.service.js]: 👍 I was having deja vu til I saw this change

File: services/github/github-issues.service.js [hunk 5]
```diff
@@ -25,81 +25,194 @@ module.exports = class GithubIssues extends LegacyService {
   static get route() {
     return {
       base: 'github',
+      pattern:
+        ':which(issues|issues-closed|issues-pr|issues-pr-closed)-:raw(raw)?/:user/:repo/:label?',
```
[the remark this answers is not in the document]
Comment (paulmelnikow) [reply services/github/github-issues.service.js]: I was really impressed that it did the right thing!

**Comment by paulmelnikow** [comment]
Good eyes! Updated :)

**Pull request merged**